src/main/java/twilightforest/entity/passive/EntityTFBighorn.java

```diff
@@ -34,7 +34,7 @@ public static int getRandomFleeceColor(Random random) {
         if (random.nextInt(2) == 0) {
             return 12;
         } else {
-            return random.nextInt(15);
+            return random.nextInt(16);
```

Comment:
tbh we could just make the return value to be `random.nextInt(16)` and remove the rest of the code in the method, having a an equal chance to encounter every color of bighorns looks funnier than just finding half of them brown.

Reply:
it's 50% of being brown and 3.33% of being any other color

Reply:
sure what ever works. Feel free to edit the pr